src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -59,6 +59,10 @@ public PersonCard(Person person, int displayedIndex) {
         address.setText(person.getAddress().value);
         email.setText(person.getEmail().value);
         matric.setText(person.getMatric().toString());
+        if (matric.getText().isEmpty()) {
```

Comment:
Nice you have edited the code such that it checks if the matric field is empty without causing any bugs for setting the text for other fields